### PR TITLE
Update Managing Content guide (SCA, Activation keys)

### PR DESCRIPTION
**Pull request by adamlazik1**
Inspect individual commits for easier review.

BZ link: https://bugzilla.redhat.com/show_bug.cgi?id=2166944

cc @jeremylenz 

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.5/Katello 4.7 (planned Satellite 6.13)
* [ ] Foreman 3.4/Katello 4.6 (EL8 only)
* [ ] Foreman 3.3/Katello 4.5 on EL7 & EL8 (Satellite 6.12 on EL8 only)
* [ ] Foreman 3.2/Katello 4.4 on EL7 & EL8
* [ ] Foreman 3.1/Katello 4.3 on EL7 & EL8 (Satellite 6.11 EL7/8, orcharhino 6.2 on EL7/8)
* For Foreman 3.0 or older, please create a separate PR.
* We do not accept PRs for Foreman 2.3 or older.

### CODE REVIEW

File: guides/common/modules/proc_creating-an-activation-key.adoc [hunk 1]
```diff
@@ -4,6 +4,14 @@
 You can use activation keys to define a specific set of subscriptions to attach to hosts during registration.
 The subscriptions that you add to an activation key must be available within the associated Content View.
 
+[IMPORTANT]
+====
+If you have Simple Content Access (SCA) enabled on {Project}, you cannot attach subscriptions to your activation key.
+With SCA enabled, subscriptions are automatically attached to your hosts.
```
Comment (jeremylenz) [inline guides/common/modules/proc_creating-an-activation-key.adoc:10]: keeping it technically accurate here :)
```suggestion
With SCA enabled, having subscriptions attached to your hosts is not needed.
```

File: guides/common/modules/snip_only-valid-if-sca-disabled.adoc [hunk 2]
```diff
@@ -0,0 +1,7 @@
+[IMPORTANT]
+====
+This procedure is only valid if you have Simple Content Access (SCA) disabled on your {Project}.
+If you have SCA enabled, subscriptions are automatically attached to your host.
```
Comment (jeremylenz) [inline guides/common/modules/snip_only-valid-if-sca-disabled.adoc:4]:
```suggestion
If you have SCA enabled, having subscriptions attached to your hosts is not needed.
```

**Review by jeremylenz — COMMENTED** [review]
Looks good except for one correction- if SCA is enabled, subscriptions aren't actually attached to the host, nor are they needed. (see below)

**Comment by jeremylenz** [comment]
Also, I think `con_managing-activation-keys.adoc` needs some updates (all the subscription-related stuff doesn't apply for SCA.)

**Comment by adamlazik1** [comment]
@jeremylenz Thank you for your feedback! Incorporated changes and ready for re-review.

Chages made:
* Reworded the notes
* Reused note about not attaching subscriptions in con_managing-activation-keys.adoc.
* Note is now in two places so made it into a snippet and put it into separate commit for easier review.

**Review by jeremylenz — APPROVED** [review]
changes LGTM, thanks! :+1:

**Comment by adamlazik1** [comment]
Does any writer want to review for wording?

**Comment by maximiliankolb** [comment]
> Does any writer want to review for wording?

I can have a look at it on Monday.

**Comment by adamlazik1** [comment]
Noted :+1:

**Review by maximiliankolb — APPROVED** [review]
LGTM

**Comment by adamlazik1** [comment]
Merged and cherry-picked:
   3e939a86f..ededdb3ae  3.5 -> 3.5

**Pull request merged**